client/python/cryoet_data_portal/src/cryoet_data_portal/_models.py

```diff
@@ -63,10 +73,18 @@ class Dataset(Model):
     cell_type_id: str = StringField()
     dataset_citations: str = StringField()
     dataset_publications: str = StringField()
+    deposition: Deposition = ItemRelationship("Deposition", "deposition_id", "id")
```

Comment:
Switching to strings instead of passing classes around is nice! I forgot that the Relationship class can handle this!